docker-entrypoint-initdb.d/001-init.sql

```diff
@@ -327640,18 +327640,18 @@ UNLOCK TABLES;
 
 DROP TABLE IF EXISTS `notification`;
 /*!40101 SET @saved_cs_client     = @@character_set_client */;
-/*!40101 SET character_set_client = utf8 */;
+/*!50503 SET character_set_client = utf8mb4 */;
 CREATE TABLE `notification` (
-  `id` int(11) NOT NULL AUTO_INCREMENT,
-  `user_id` bigint(20) NOT NULL,
+  `id` int NOT NULL AUTO_INCREMENT,
+  `user_id` bigint NOT NULL,
   `seen` tinyint(1) NOT NULL DEFAULT '0',
   `date` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
   `email` tinyint(1) NOT NULL,
```

Comment:
still looking intact after changing MySQL version

Reply:
The problem was at first place that the sqlx.json wasn't updated. But we also added a default value, as it is in production.